README.md

```diff
@@ -1,8 +1,73 @@
 # orb_slam_2_ros
-This package integrates orb_slam_2 into ROS in what I believe to be a more user friendly way than what is offered by the underlying library found here:
 
-https://github.com/raulmur/ORB_SLAM2
+## Overview
 
-Depends on my catkinized version of the orb_slam_2 library:
+This package integrates orb_slam_2 into ROS in a more user friendly way than what is offered by the original library [ORB_SLAM2](https://github.com/raulmur/ORB_SLAM2)
 
-https://github.com/ethz-asl/orb_slam_2_catkin
+**Keywords:** ROS, ORB_SLAM2, SLAM
+
+### License
+
+The source code is released under a [TODO @alexmillane insert license](put_link_here).
+
+**Author(s): Alex Millane
+Maintainer: Alex Millane
+Affiliation: Autonomous Systems Lab, ETH Zurich**
+
+The orb_slam_2_ros package has been tested under ROS Indigo on Ubuntu 14.04 and under ROS Kinetic Ubuntu 16.04. This is research code, expect that it changes often and any fitness for a particular purpose is disclaimed.
+
+
+## Installation: building from Source
+
+### Dependencies
+Make sure you have added your SSH keys to your Github account. For more info check [connecting-to-github-with-ssh](https://help.github.com/articles/connecting-to-github-with-ssh/).
+
+- [Robot Operating System (ROS)](http://wiki.ros.org) (middleware for robotics),
+- [catkin_simple](https://github.com/catkin/catkin_simple)
+	```
+	cd ~/catkin_workspace/src
+	git clone git@github.com:catkin/catkin_simple.git # cloning over SSH (github account needed)
+	# git clone https://github.com/catkin/catkin_simple.git # cloning over HTTPS (no github account needed)
+	```
+- [catkinized version of the orb_slam_2](https://github.com/ethz-asl/orb_slam_2_catkin)
+	```
+	cd ~/catkin_workspace/src
+	git clone git@github.com:ethz-asl/orb_slam_2_catkin.git # cloning over SSH (github account needed)
+	# git clone https://github.com/ethz-asl/orb_slam_2_catkin.git # cloning over HTTPS (no github account needed)
+	```
+- [image_undistort](https://github.com/ethz-asl/image_undistort)
+	```
+	cd ~/catkin_workspace/src
+	git clone git@github.com:ethz-asl/image_undistort.git # cloning over SSH (github account needed)
+	# git clone https://github.com/ethz-asl/image_undistort.git # cloning over HTTPS (no github account needed) 
+	```
+- [pangoli_catkin](https://github.com/ethz-asl/pangolin_catkin)
```

Comment:
"pangoli" -> "pangolin"

I think that pangolin is actually an rpg package with the link being:

https://github.com/uzh-rpg/pangolin_catkin

Reply:
good catch!